# The Experiment

Uropathogenic _Escherichia coli_ (UPEC) employs a complex set of molecular factors that allow it to evade host defences, resist antibiotic treatment, and persist in the urinary tract, ultimately facilitating pathogenesis.

To remain in the urinary tract UPEC employs _adherence factors_, molecules that act to help UPEC adhere to the tissue of the channel itself, or to material in that channel, such as catheter tubes. If we can intervene to reduce this adherence, we can shorten the dwell time of the microbe and reduce the impact of disease.

## Experimental overview

The overall plan of the experiment is sketched in @fig-experiment.

![Overview of the experimental protocol. 1: a BAC library is constructed for UPEC in a DH10B _E.coli_ background. 2: a triplicate library of 384clones is produced. The library is passaged three times through (3a) a control arm or (3b) a mouse/catheter model. 4: After serial passage, total gDNA from the resulting cultures is Illumina sequenced.](assets/images/experiment.png){#fig-experiment width=80% .lightbox}

A bacterial artificial chromosome (BAC) library is constructed for an _E. coli_ UPEC isolate.

::: { .callout-tip title="Bacterial artificial chromosomes (BACs)" }
A bacterial artificial chromosome is a DNA construct that introduces a large genomic region (typically 150-350kbp) from a genome of interest into a "neutral" background genome. In this experiment, genomic regions from UPEC are introduced into a "neutral" background _E. coli_ (DH10B). This allows us to investigate the effect of genes from UPEC when expressed by an otherwise non-pathogenic _E. coli_.
:::

::: { .callout-note collapse="true" title="Experimental detail" }
The UPEC genome was partially digested with _Hind_III and cloned into a pV41 vector. This was used to generate three pools of 384 BAC clones to facilitate replication.
:::

The BAC library is then passaged through a mouse model (catheterised mouse) to emulate introduction to the human catheterised urinary tract. After passage and washing of the extracted catheter tube, the tube was used as inoculum for a further round of incubation to culture adherent bacteria.

::: { .callout-tip title="Passage" }
Passage is the serial, iterated growth of bacteria under a particular environment or series of environments. This approach is usually taken to subject the bacteria to a specific selection pressure so that we can observe how it evolves in response to that pressure.

This is a relatively common approach to investigating bacterial virulence factors.
:::

::: { .callout-note collapse="true" title="Experimental detail" }
Mouse passage was allowed to proceed for 18h. Subsequent culturing was performed in LB media at 37 degC for 18h between passages.

A parallel control passage, identical except that bacteria were cultured in LP for 18h rather than in the mouse model, was also performed.

Three rounds of passage were performed.
:::

After passage, the total gDNA from the three control and three mouse system cultures is sequenced. This produces read counts corresponding to three kinds of genes:

1. genes present only in the background DH10B
2. genes present in both DH10B and UPEC
3. genes present only in UPEC and carried through passage

## Expected interpretation

We are most interested in the genes that are present only in UPEC, but which form part of the BAC library.

We expect that the serial passage process enriches our cultures for bacteria that adhere better to the urinary tract and/or catheter material. As individual clones contain only fragments of the UPEC genome, those clones that don't contain adherence factors are expected to be selected out of the mixture, leaving only clones with UPEC genes that promote adherence.

By statistical analysis of the read counts, we aim to identify UPEC-specific genes that are overrepresented in the final mixtures that have been through mouse passage, but that are not overrepresented in the control passage through culture alone.

::: { .callout-caution title="How did that work out?"}
You can see how well this worked, if at all, in the next section.
:::